cli/lesspass/core.py

```diff
@@ -30,7 +31,11 @@ def main(args=sys.argv[1:]):
         sys.exit(4)
 
     if not args.master_password:
-        args.master_password = getpass.getpass("Master Password: ")
+        prompt = "Master Password: "
+        if not args.fingerprint:
```

Comment:
reverse your `if not else` into `if else`

Reply:
resolved in 66b481c